datalad/support/annexrepo.py

```diff
@@ -2761,12 +2776,7 @@ def is_available(self, file_, remote=None, key=False, batch=False):
         annex_input = [key_,] if not remote else [key_, remote]
 
         if not batch:
-            try:
-                out = self.call_annex(['checkpresentkey'] + annex_input)
-                assert(not out)
```

Comment:
not sure how important but we are loosing now this `assert`ed behavior since in call_annex_success there is no check of output.  It was originally added in the initial implementation in c5ba8b6baa24c379ae41dff0018b152196d243b7 without any special comment on its necessity etc, and we never triggered it AFAIK.  I guess it could be sacrificed.

Reply:
This is what the manpage of this command says about its output:

> Exits 0 if the content is verified present in the remote, or 1 if it is
       verified to not be present in the remote. If there is  a  problem,  the
       special  exit  code  100  is  used,  and  an error message is output to
       stderr.

Reply:
FTR: I support not relying on output assumptions.